.github/workflows/minimal-tests.yml

```diff
@@ -29,7 +29,9 @@ jobs:
       - name: Install Requirements
         run: |
           python -m pip install --upgrade pip
-          pip install -U -r requirements-dev.txt
+          python -m pip install poetry==1.1.7
```

Comment:
1. It looks like installing via pip is discouraged by [poetry](https://python-poetry.org/docs/#installing-with-pip) or is that not important?
2. Any reason for freezing to 1.1.7?

Reply:
1. From my experience, installing `poetry` via the curl command has its benefits if you install it on your local system but in CI/CD it does not make a huge difference. Only point might be if at some point `poetry` and `awswrangler` depend on the package but with different versions.
2. I've just pinned it to the latest version, as in the past an update introduced some issues but as it's only in CI/CD we could also unpin it.